### PR TITLE
Implements responsive banner slider

### PR DESCRIPTION
**Pull request by amoreira**

### CODE REVIEW

File: src/components/Banner/index.js [hunk 1]
```diff
@@ -6,29 +6,67 @@ import Slider from 'react-slick';
...
 class Banner extends React.Component {
   constructor(props) {
     super(props);
-    this.state = this.props.settings ? this.props.settings.toJS() : {};
+    const settings = this.props.settings ? this.props.settings.toJS() : {};
+    this.state = { ...settings, viewport: getViewPort()};
+  }
+
+  componentDidMount = () => {
+    window.addEventListener('resize', this.onResize);
```
Comment (brenoc) [inline src/components/Banner/index.js]: Listening to resize or scroll events like that are really cost **in**efficient. Imagine this function being called 300 times when the user resize the browser window in a 2 second timeframe.

It's a good practice to add a debounce or throttle here. 

If you want a good read about this, I recomend this: http://drupalmotion.com/article/debounce-and-throttle-visual-explanation

Comment (amoreira) [inline src/components/Banner/index.js]: good to know!

**Comment by amoreira** [comment]
@brenoc done!

**Comment by brenoc** [comment]
LGTM :+1:

**Comment by brenoc** [comment]
@tamorim will you look at that?

**Pull request merged**